src/Form/Type/ModelAutocompleteType.php

```diff
@@ -42,12 +42,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->setAttribute('minimum_input_length', $options['minimum_input_length']);
         $builder->setAttribute('items_per_page', $options['items_per_page']);
         $builder->setAttribute('req_param_name_page_number', $options['req_param_name_page_number']);
-        $builder->setAttribute(
-            'disabled',
-            $options['disabled']
-            // NEXT_MAJOR: Remove this when bumping Symfony constraint to 2.8+
-            || (array_key_exists('read_only', $options) && $options['read_only'])
-        );
+        $builder->setAttribute('disabled', $options['disabled']);
```

Comment:
Why did we change this one?

IMO this PR should only remove code that was never executed because of the conditionals. Not sure about changing this behavior

Reply:
`read_only` is deprecated but still can be specified in Symfony 2.8. Reverted.
Thanks @jordisala1991